ios/RNCCameraRollManager.m

```diff
@@ -137,6 +137,11 @@ static void requestPhotoLibraryAccess(RCTPromiseRejectBlock reject, PhotosAuthor
       PHAssetChangeRequest *assetRequest ;
       if ([options[@"type"] isEqualToString:@"video"]) {
         assetRequest = [PHAssetChangeRequest creationRequestForAssetFromVideoAtFileURL:inputURI];
+      } else if ([inputURI.pathExtension isEqualToString:@"gif"]) {
```

Comment:
suggest inputURL.toLowerString, because some file like aaa.GIF

Reply:
@sergeymild would you update to lowerCase to avoid having issue, and then we can merge this PR